lib/takeaway.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'food_menu.rb'
+require_relative 'order_management.rb'
+require_relative 'twilio_message.rb'
+
+class Takeaway
+
+  attr_reader :food_menu, :order_management, :message
+
+  def initialize(food_menu, order_management = OrderManagement.new, message = Message.new)
+    @food_menu = food_menu
+    @order_management = order_management
+    @message = message
+  end
+
+  def see_dishes
+    food_menu.print
+  end
+
+  def place_the_order(selected_dishes)
```

Comment:
There was a requirement to pass in a total value here which should return an error if the total didn't match the calculated total. This could be something to add if you revisit the code